df_py/util/dftool_module.py

```diff
@@ -681,14 +700,27 @@ def do_dispense_active():
     from_account = _getPrivateAccount()
     token_symbol = B.Simpletoken.at(TOKEN_ADDR).symbol().upper()
     token_symbol = token_symbol.replace("MOCEAN", "OCEAN")
-    rewards = csvs.loadRewardsCsv(CSV_DIR, token_symbol)
 
-    # "flatten" the rewards dict to dispense all chains in one go
-    all_rewards = calcrewards.flattenRewards(rewards)
+    rewards = {}
```

Comment:
It's weird to start with an empty `rewards` dict then update it with subsequent calls to calc rewards for each substream.

Rather, each substream should calc its own rewards from its budget. 

It'd look something like:
```python
volume_rewards = load_volume_rewards_csv(...)
predictoor_rewards = load_predictoor_rewards_csv(...)
challenge_rewards = load_challenge_rewards_csv(...)
rewards = merge_rewards(volume_rewards, predictoor_rewards, challenge_rewards)
```

----

In an earlier comment I mention how we don't need to specify "OCEAN" token in rewards filenames anymore. Therefore we don't need "OCEAN" arguments in the function calls below.